app/models/event.rb

```diff
@@ -93,4 +97,14 @@ def editable_by?(user)
   def uneditable_by?(user)
     !editable_by?(user)
   end
+
+  def self.number_of_events_per_country(country_rank)
```

Comment:
countries = Event.all.group_by(&:country)
sorted_events = countries.sort_by{|country, events| events.count}
// this gives a nested hash: of [[country, events], [country, events]
// sorted_events[-1][0] would give us the name of the country with most events
// sorted_events[-1][1].count would give us the number of events

Reply:
sorted_events[-1].first
sorted_events[-1].last.count